<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1136</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QwtPlot" name="plotSignal">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>20</y>
      <width>1101</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QwtPlot" name="plotSpectrum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>1101</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>490</y>
      <width>981</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
